themes/default/content/blog/pulumi-release-notes-0.57/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Pulumi release notes: Milestone 58"
+date: 2021-06-08T11:47:31-07:00
+draft: false
+meta_desc: "New in Pulumi: Amazon ECS Anywhere and App Runner, Sumo Logic support, Pulumi Packages updates, and easier updates for teammates in the Pulumi Service"
+meta_image: meta.png
+authors:
+    - alex-mullans
+tags:
+    - features
+---
+
+We release new versions of the Pulumi CLI every two weeks, new versions of the Pulumi Service multiple times a week, and new and updated cloud providers frequently. To keep up with all of these changes, you used to have to find the `CHANGELOG.md` files in the repo (or repos!) that you care about, which could get unwieldy. So, we're trying something new: a tri-weekly rollup of Pulumi news, features, and updates. If you find it useful, or have ideas on how we could improve it, let us know [on Twitter](https://twitter.com/pulumi) or in the [Pulumi Community Slack](https://slack.pulumi.com)!
```

Comment:
```suggestion
We release new versions of the Pulumi CLI every two weeks, new versions of the Pulumi Service multiple times a week, and new and updated cloud providers frequently. To keep up with all of these changes, you used to have to find the `CHANGELOG.md` files in the repo (or repos!) that you care about, which could get unwieldy. So, we're trying something new: a tri-weekly rollup of Pulumi release news, features, and updates. If you find it useful, or have ideas on how we could improve it, let us know [on Twitter](https://twitter.com/pulumicorp) or in the [Pulumi Community Slack](https://slack.pulumi.com)!
```

Also wonder if we can avoid the negative aspect of this phrasing?  Highlight that all realase notes are avialble in CHANGELOGs already today, but that we often want to highlight more details than we can in the CHANGELOG, and to highlight key new features across the dozens of repositories that make up the Pulumi open source project.

Reply:
Updated, LMK what you think